wbooks-api/db/migrate/20180829144126_add_first_name_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddFirstNameToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :first_name, :string, null: false
```

Comment:
if first_name is not nullable you should validate the presence in the user model